pkg/occlient/occlient_test.go

```diff
@@ -749,7 +750,7 @@ func TestCreateRoute(t *testing.T) {
 			service:    "mailserver",
 			portNumber: intstr.FromInt(8080),
 			labels: map[string]string{
-				"SLA":                              "High",
```

Comment:
Isn't that going against what `go fmt` prescribes?

Reply:
Yeah, I just realized that, it seems that `goimports` isn't following standard gofmt conventions...